src/diffusers/loaders.py

```diff
@@ -0,0 +1,177 @@
+# Copyright 2022 The HuggingFace Team. All rights reserved.
```

Comment:
The idea is to gather all the special loaders in this file, such as Textual Inversion, Hypernetworks, ...